.github/workflows/tests.yaml

```diff
@@ -1,30 +1,28 @@
 name: Tests
 on: [push, pull_request]
 jobs:
-
```

Comment:
Why do we need to remove that? Can we keep the semantic space that helps in reading and visualizing blocks?